frontend/__tests__/utils/check-hardcoded-strings.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { render } from "@testing-library/react";
+import { test, expect, describe, vi } from "vitest";
+import { InteractiveChatBox } from "#/components/features/chat/interactive-chat-box";
+import { ChatInput } from "#/components/features/chat/chat-input";
+
+// Mock react-i18next
+vi.mock("react-i18next", () => ({
+  useTranslation: () => ({
+    t: (key: string) => key,
+  }),
+}));
+
+describe("Check for hardcoded English strings", () => {
+  test("InteractiveChatBox should not have hardcoded English strings", () => {
+    const { container } = render(
+      <InteractiveChatBox
+        onSubmit={() => {}}
+        onStop={() => {}}
+      />
+    );
+
+    // Get all text content
+    const text = container.textContent;
+
+    // List of English strings that should be translated
+    const hardcodedStrings = [
+      "What do you want to build?",
+    ];
+
+    // Check each string
+    hardcodedStrings.forEach(str => {
+      expect(text).not.toContain(str);
+    });
```

Comment:
```suggestion
    render(<InteractiveChatBox onSubmit={() => {}} onStop={() => {}} />);
    expect(screen.queryByText("What do you want to build?")).not.toBeInTheDocument();
```

It is not necessary to introduce an array since it contains a single value only, and I doubt this test will be extended in the foreseeable future.